### PR TITLE
Add alerts and modify the dashboards

### PR DESCRIPTION
**Pull request by zhouzhuojie**
Steps to add or change grafana dashboards:

- Go to https://metrics.pytorch.org
- Edit the dashboards, or alerts directly within the dashboard
- Go to the settings page, and save the JSON
- Setup a PR and commit

This PR adds 3 alerts
- squid-proxy unhealth instances count (if >10 within 5min at 1min interval check)
- pytorch-probot errors count (if >10 within 5min at 1min interval check)
- scribe-proxy errors count (if >10 within 5min at 1min interval check)

### CODE REVIEW

File: aws/websites/metrics.pytorch.org/files/dashboards/squid_proxy_status.json [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+  "templating": {
+    "list": []
+  },
+  "time": {
+    "from": "now-6h",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "",
+  "title": "Squid Cached Proxy Copy",
```
Comment (driazati) [inline aws/websites/metrics.pytorch.org/files/dashboards/squid_proxy_status.json:465]:
```suggestion
  "title": "Squid Cached Proxy",
```

File: aws/websites/metrics.pytorch.org/files/dashboards/squid_proxy_status.json [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+  "uid": "4xOx_sZnz",
+  "version": 3
+}
```
Comment (malfet) [inline aws/websites/metrics.pytorch.org/files/dashboards/squid_proxy_status.json:468]: Nit: newline
```suggestion
}
```

**Review by malfet — APPROVED** [review]

**Review by driazati — APPROVED** [review]
nice! It's probably worth it to add those steps to [the README](https://github.com/pytorch/test-infra/tree/master/aws/websites/metrics.pytorch.org#adding-a-dashboard)

image

The colors for `gh-ci-scale-down Invocations and Errors` are flipped, can you fix it so they match the others?

**Comment by zhouzhuojie** [comment]
Looking into the colors, it's consistent on my end, maybe we need to explicitly assign the colors, one moment for incoming changes

**Comment by zhouzhuojie** [comment]
Updated

image

**Pull request merged**